### DISCUSSION TITLE
DND modifiers for each item to work on resize or drag dyamically.

### DISCUSSION THREAD

**Discussion by khanzzirfan** [samuelarbibe/dnd-timeline#54] [Q&A]
I was thinking to have resize item with max and min time limits. so that user can resize the item within the max and min boundaries. 
I resize should not go beyond the max and min values set for the item. 

is there way we can achieve this right now? I had a look at the dnd-kit , the provide resize modifiers, specially for DragOverlay items. I would like to know if this is something can be achieveable.  if so, please provide me some guidance how to do it. 

An example
```
function App() {
  return (
    <DndContext modifiers={[restrictToVerticalAxis]}>
      {/* ... */}
      <DragOverlay modifiers={[restrictToWindowEdges]}>
        {/* ... */}
      </DragOverlay>
    </DndContext>
  )
}
``` 
In this example I would like dnd-timeline , modifiers to work with a range set (min , max) so that it can be resize within the given range limit.

**Comment by samuelarbibe** [comment]
Hey!
The resize implementation is internal and not related to dnd-kit.

As of today, there is no way to limit the onResizeMove event, but only limit onDragEnd event.

If you are interested in a limit mechanism that work on the drag animation itself, while resizing, Please open an issue and we can talk about a solution 🤝
